bsdlib/include/nrf_socket.h

```diff
@@ -1264,7 +1300,7 @@ int nrf_select(int                        nfds,
  * @brief Method to poll for events on one or more sockets.
  *
  * @param[inout] An array of sockets, and respective for each socket that the caller polls for.
- *               The occurred events per socket is returned in the requested field of @ref struct nrf_pollfd structure.
+ *               The occurred events per socket is returned in the revents field of @ref struct nrf_pollfd structure.
  *               Shall not be NULL.
  * @param[in]    Positive number of sockets being polled for events.
```

Comment:
```suggestion
 * @param[in] nfds    Positive number of sockets being polled for events.
```

Also below ( I can't comment there as it's outside of PR scope):
1. In line 1308: ``@param[in] timeout   Timeout in milliseconds.``
2. In line 1311: ``@return A positive number less than or equal to nfds indicating sockets on which events occurred.``

Reply:
ok